src/Node/Api.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the KongAdminApi package.
+ *
+ * (c) Unikorp <https://github.com/unikorp>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Unikorp\KongAdminApi\Node;
+
+use Unikorp\KongAdminApi\AbstractNode;
+use Unikorp\KongAdminApi\Document\Api as Document;
+
+/**
+ * api
+ *
+ * @author VEBER Arnaud <https://github.com/VEBERArnaud>
+ */
+class Api extends AbstractNode
+{
+    /**
+     * add api
+     *
+     * @param \Unikorp\KongAdminApi\Document\Api $document
```

Comment:
Node and Document classes has the same name (`Api` in this case), that's why I am using an alias (`use \Unikorp\KongAdminApi\Document\Api as Document`)

In the docblock I prefer to use the FQCN even if the function signature reference `Document`